### PR TITLE
fixed CQ for bitmessageqt.messagecompose module

### PR DESCRIPTION
**Pull request by coolguy-cell**
flake8 fixes
pycodestyle fixes
pylint fixes

### CODE REVIEW

File: src/bitmessageqt/messagecompose.py [hunk 1]
```diff
@@ -1,23 +1,41 @@
+"""
```
Comment (PeterSurda) [inline src/bitmessageqt/messagecompose.py:1]: Message editor module with a wheel zoom functionality.

File: src/bitmessageqt/messagecompose.py [hunk 2]
```diff
@@ -1,23 +1,41 @@
...
+# pylint: disable=bad-continuation
+
 from PyQt4 import QtCore, QtGui
 
+
 class MessageCompose(QtGui.QTextEdit):
-    
-    def __init__(self, parent = 0):
+    """MessageCompose class for Qt UI"""
```
Comment (PeterSurda) [inline src/bitmessageqt/messagecompose.py:12]: Editor class with wheel zoom functionality.

File: src/bitmessageqt/messagecompose.py [hunk 3]
```diff
@@ -1,23 +1,41 @@
...
-        self.setAcceptRichText(False) # we'll deal with this later when we have a new message format
+        self.setAcceptRichText(False)
+        # we'll deal with this later when we have a new message format
         self.defaultFontPointSize = self.currentFont().pointSize()
-    
+
     def wheelEvent(self, event):
-        if (QtGui.QApplication.queryKeyboardModifiers() & QtCore.Qt.ControlModifier) == QtCore.Qt.ControlModifier and event.orientation() == QtCore.Qt.Vertical:
+        """Mouse wheel scroll event handler"""
+        # a special accept flag that indicates whether the receiver wants the event
```
Comment (PeterSurda) [inline src/bitmessageqt/messagecompose.py:21]: why the comment

File: src/bitmessageqt/messagecompose.py [hunk 4]
```diff
@@ -1,23 +1,41 @@
...
+                QtGui.QApplication.translate("MainWindow", "Zoom level %1%").arg(
+                    str(zoom)
+                )
+            )
         else:
             # in QTextEdit, super does not zoom, only scroll
             super(MessageCompose, self).wheelEvent(event)
 
     def reset(self):
+        """reset method for setting text"""
```
Comment (PeterSurda) [inline src/bitmessageqt/messagecompose.py:40]: `"""Clear the edit content"""`

**Comment by coolguy-cell** [comment]
fixed

File: src/bitmessageqt/messagecompose.py (hunk 1 above)
Comment (coolguy-cell) [reply src/bitmessageqt/messagecompose.py:1]: Done

File: src/bitmessageqt/messagecompose.py (hunk 3 above)
Comment (coolguy-cell) [reply src/bitmessageqt/messagecompose.py]: removed.

File: src/bitmessageqt/messagecompose.py (hunk 4 above)
Comment (coolguy-cell) [reply src/bitmessageqt/messagecompose.py]: Done.

File: src/bitmessageqt/messagecompose.py (hunk 2 above)
Comment (coolguy-cell) [reply src/bitmessageqt/messagecompose.py]: Done

File: src/bitmessageqt/messagecompose.py [hunk 5]
```diff
@@ -1,23 +1,39 @@
+"""
+Message editor with a wheel zoom functionality
+
```
Comment (PeterSurda) [inline src/bitmessageqt/messagecompose.py:3]: Why is line 3 empty?

File: src/bitmessageqt/messagecompose.py [hunk 6]
```diff
@@ -1,23 +1,39 @@
...
 class MessageCompose(QtGui.QTextEdit):
-    
-    def __init__(self, parent = 0):
+    """Editor class with wheel zoom functionality"""
+    def __init__(self, parent=0):
         super(MessageCompose, self).__init__(parent)
-        self.setAcceptRichText(False) # we'll deal with this later when we have a new message format
+        self.setAcceptRichText(False)
+        # we'll deal with this later when we have a new message format
```
Comment (PeterSurda) [inline src/bitmessageqt/messagecompose.py:15]: Remove like 15 (the comment)

File: src/bitmessageqt/messagecompose.py (hunk 5 above)
Comment (coolguy-cell) [reply src/bitmessageqt/messagecompose.py:3]: Oh, yes. I totally forget to remove it.

**Comment by coolguy-cell** [comment]
Done.

**Pull request merged**